app_gui

```diff
@@ -0,0 +1,406 @@
+from PyQt5.QtWidgets import (QApplication, QWidget, QVBoxLayout, QLabel, QLineEdit, 
+                             QPushButton, QMessageBox, QComboBox, QHBoxLayout)
+import sys
+from modules.StudentManagemnt import *
+from modules.WorkerManagement import *
+
+class UniversityManagementApp(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.initUI()
+        
+    def initUI(self):
+        self.setGeometry(100, 100, 400, 300)
+        self.setWindowTitle('University Management Application')
+        
+        layout = QVBoxLayout()
+
+        # Welcome Label
+        self.welcome_label = QLabel("Hello, Welcome to University Management Application", self)
+        layout.addWidget(self.welcome_label)
+        
+        # ComboBox for Management Selection
+        self.combo_box = QComboBox(self)
+        self.combo_box.addItem("Select Management")
+        self.combo_box.addItem("Student Management")
+        self.combo_box.addItem("Worker Management")
+        layout.addWidget(self.combo_box)
+        
+        # Button to Manage
+        self.manage_button = QPushButton("Manage", self)
+        self.manage_button.clicked.connect(self.manage)
+        layout.addWidget(self.manage_button)
+        
+        self.setLayout(layout)
+    
+    def manage(self):
+        choice = self.combo_box.currentIndex()
+        if choice == 1:
+            self.student_window = StudentManagement(self)
+            self.student_window.show()
+            self.hide()
+        elif choice == 2:
+            self.worker_window = WorkerManagement(self)
+            self.worker_window.show()
+            self.hide()
+        else:
+            QMessageBox.warning(self, "Error", "Please select a valid option")
+
+class StudentManagement(QWidget):
+    def __init__(self, parent):
+        super().__init__()
+        self.parent = parent
+        self.initUI()
+        
+    def initUI(self):
+        self.setGeometry(100, 100, 400, 300)
```

Comment:
Those are the same dimensions used above that's why you should declare them as them as global variables